plugins/core/tracer.go

```diff
@@ -65,6 +65,10 @@ func (t *Tracer) Init(entity *reporter.Entity, rep reporter.Reporter, samp Sampl
 	t.Reporter.Boot(entity, t.cdsWatchers)
 	t.initFlag = 1
 	t.initMetricsCollect(meterCollectSecond)
+	t.correlation = &CorrelationConfig{
```

Comment:
Can we make it configurable?

Reply:
looks like it has not been fixed?

Reply:
sorry,I misunderstood it as supplementing comments on GitHub.I will resolve it.